src/store.js

```diff
@@ -24,16 +26,37 @@ export default new Vuex.Store({
         .auth()
         .createUserWithEmailAndPassword(email, password)
         .then(() => {
-          
+          db.collection("users").add({
+            name: this.state.name,
+            email: email,
+            password: password
+        })
         })
         .catch(error => {
           console.error('Account Regeister Error', error.message);
         });
     },
+    userSignIn(state, { email, password}) {
```

Comment:
{}の内側のスペースがバラバラになっています。
前後を開ける形で統一すると良いです。
```suggestion
    userSignIn(state, { email, password }) {
```
これも自動整形でできます。